source/SIL.AppBuilder.Portal.Frontend/src/data/with-current-user.tsx

```diff
@@ -69,7 +69,7 @@ export function withCurrentUser() {
         // NOTE: this whole lifecycle hook is kind of a hack for lack
         //       of a better 'get the current user' pattern.
         if (fromCache && fromCache.attributes && fromCache.attributes.auth0Id === auth0IdFromJWT) {
-          this.setState({ currentUser: fromCache });
+          this.setState({ currentUser: fromCache, isLoading: false });
```

Comment:
@giancorzo this fixes the issue you discovered on friday where navigating back and forth to a few places shows the loader.

Reply:
Cool